sherpa/astro/datastack/tests/test_datastack.py

```diff
@@ -680,7 +680,7 @@ def test_case_7(self):
 
         assert f == [1, 2]
 
-        f = datastack.query_by_obsid('7867')
+        f = datastack.query_by_obsid(7867)
```

Comment:
I just realized we didn't have a test for the integer case, so I changed one of the tests. A string is tested before and after this test. Or am I missing something?